packages/design-system/src/components/VerticalNav/VerticalNavItemLabel.test.tsx

```diff
@@ -3,7 +3,7 @@ import VerticalNavItemLabel from './VerticalNavItemLabel';
 import { shallow } from 'enzyme';
 
 function shallowRender(customProps = {}) {
-  const props = Object.assign(
+  const props: any = Object.assign(
```

Comment:
I think it would work to type this as `Partial<VerticalNavItemLabelProps>` if `customProps` also had that type